File: platforms/kotlin/examples/core/renderer/create_texture_with_size.kt
import org.fragmentcolor.*
val renderer = Renderer()
val pixels = [
    255,0,0,255,   0,255,0,255,
    0,0,255,255,   255,255,255,255,
]
val tex = renderer.createTextureWithSize(pixels, arrayOf(2, 2))

File: platforms/python/examples/core/renderer/create_texture_with_size.py
from fragmentcolor import Renderer, Size
renderer = Renderer()
pixels = [
    255,0,0,255,   0,255,0,255,
    0,0,255,255,   255,255,255,255,
]
tex = renderer.create_texture_with_size(pixels, [2, 2])